apps/pwabuilder/src/script/pages/app-report.ts

```diff
@@ -1617,18 +1716,93 @@ export class AppReport extends LitElement {
+    this.isAppCardInfoLoading = true;
+
+    let findersResults = {
+      manifest: {} as {url?: string, raw?: string, json?: unknown},
+      serviceWorker: {} as {url?: string, raw?: string}
+    }
+
+    // Take only good results, ignore errors.
+    FindWebManifest(url).then( async (result) => {
+      if (result?.content?.raw) {
+        // TODO: can use json instead of raw
+        findersResults.manifest = result.content;
+        await this.applyManifestContext(url, result?.content?.url || undefined, result?.content?.raw);
+        await this.testManifest();
+      }
+    });
+    // .catch(async () => {
+    //   await this.applyManifestContext(url, undefined, undefined);
+    // }).finally(async () => {
+
+    //   await this.testManifest();
+    // });
+
+    FindServiceWorker(url).then( async (result) => {
+        if (result?.content?.url) {
+          await AuditServiceWorker(result.content.url).then( (result) => {
+            this.testServiceWorker(processServiceWorker(result.content, undefined));
+          });
+          findersResults.serviceWorker = result.content;
+        }
+        // else {
+        //   this.testServiceWorker(processServiceWorker({score: false, details: {}}, false));
+        // }
+      }
+    );
+    // .catch(() => {
+    //   this.testServiceWorker(processServiceWorker({score: false, details: {}}, false));
+    // });
+
+    try {
+      this.reportAudit = await Report(url);
+    } catch (e) {
+      console.error(e);
+      await this.testSecurity(processSecurity());
+      if (!findersResults.manifest?.raw) {
+        await this.applyManifestContext(url, undefined, undefined);
+        await this.testManifest();
+      }
+      if (!findersResults.serviceWorker?.raw) {
+        this.testServiceWorker(processServiceWorker({score: false, details: {}}, false));
+      }
+      this.runningTests = false;
+      return;
+    }
+
+    console.log(this.reportAudit);
+
+    if (this.reportAudit?.artifacts?.webAppManifest) {
+      await this.applyManifestContext(url, this.reportAudit?.artifacts?.webAppManifest?.url, this.reportAudit?.artifacts?.webAppManifest?.raw);
+    } else {
+      await this.applyManifestContext(url, undefined, undefined);
+    }
+
+    this.todoItems = [];
+    // await this.getManifest(url);
+    await Promise.all([
+      this.testManifest(),
+      this.testServiceWorker(processServiceWorker(this.reportAudit?.audits?.serviceWorker, this.reportAudit?.audits?.installableManifest?.score)),
+      this.testSecurity(processSecurity(this.reportAudit?.audits))]).then(() =>
     {
       //this.canPackage = this.canPackageList.every((can: boolean) => can);
       // this.canPackageList: boolean[] = [canPackageManifest?, canPackageSW?, canPackageSec?]
       this.canPackage = this.canPackageList[0] && this.canPackageList[2];
     });
 
-    await this.testServiceWorker(url);
+    // await this.testServiceWorker(url);
```

Comment:
So the fix to the above would be to remove the SW test in L1798 and paste it here